### PR TITLE
doc: add minutes for meeting Oct 31

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2018-10-31.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  * Peter Marshall (https://github.com/nodejs/node/issues/23849)
+  * Ouyang Yadong (https://github.com/nodejs/node/issues/23955)
+  * Masashi Hirano (https://github.com/nodejs/node/issues/23850)
+
+### nodejs/node
+
+* fs: behaviour of readFile and writeFile with file descriptors [#23433](https://github.com/nodejs/node/issues/23433)  
+  * Vote has been suggested.
+  * Michael have we had complaints from end users
+    * Sakthipriyan - believe just raised by collaborators
```
Comment (sam-github) [inline meetings/2018-10-31.md]: @jorangreef is not a collaborator, and originally raised  https://github.com/nodejs/node/issues/9671
@thisconnect and @bzoz from https://github.com/nodejs/node/issues/13572 are not collaborators

Comment (richardlau) [reply meetings/2018-10-31.md]: @bzoz is a collaborator.

File: meetings/2018-10-31.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  * Matteo hat this been assessed by Canary in the Goldmine
```
Comment (richardlau) [inline meetings/2018-10-31.md]:
```suggestion
  * Matteo has this been assessed by Canary in the Goldmine
```

File: meetings/2018-10-31.md (hunk 1 above)
Comment (thefourtheye) [reply meetings/2018-10-31.md]: @sam-github Ah, thanks for correcting my understanding.

File: meetings/2018-10-31.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  * Masashi Hirano (https://github.com/nodejs/node/issues/23850)
+
+### nodejs/node
+
+* fs: behaviour of readFile and writeFile with file descriptors [#23433](https://github.com/nodejs/node/issues/23433)  
+  * Vote has been suggested.
+  * Michael have we had complaints from end users
+    * Sakthipriyan - believe just raised by collaborators
+  * Matteo has this been assessed by Canary in the Goldmine
+    * Not yet seomething we should do
```
Comment (thefourtheye) [inline meetings/2018-10-31.md]: Actually, CITGM was tried for the first option's PR (https://github.com/nodejs/node/pull/23709), but the failing tests were already failing for sometime.

File: meetings/2018-10-31.md (hunk 1 above)
Comment (thisconnect) [reply meetings/2018-10-31.md]: > behavior of readFile and writeFile with file descriptors

Thank you all for looking into this and taking action! It doesn't matter which direction you choose, everything is better than current situation. There only mistake would be to do nothing. :)

File: meetings/2018-10-31.md (hunk 3 above)
Comment (mhdawson) [reply meetings/2018-10-31.md]: @thefourtheye updated to reflect that.

**Pull request merged**